content/en/storage-provider/architecture/sp-automation/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Lotus automation"
-description: "1-click deployment automation for Lotus allows new storage providers to quickly learn and deploy Lotus."
+title: "SP automation"
```

Comment:
```suggestion
title: "Storage provider automation"
```

Can we change it to this? Having an acronym in the sidebar is bad UX.

If not, how about _Automated setup_ or something like that?